tasks/12.txt

```diff
@@ -3,3 +3,13 @@ Write a SQL query to get Product list of twenty most expensive products:
 | ProductName | UnitPrice |
 
 Order products by price.
+
+db.products.aggregate([
+    { 
+        $sort: { UnitPrice: -1 } 
+    },
+    { 
+        $project: { _id: 0, ProductName:1, UnitPrice:1 } 
+    },
+    { $limit : 20 }
+])
```

Comment:
It is best to use $limit as soon as you're able to.
In your case $project will be applied for all documents in products and only after that $limit